modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/WindowStage.java

```diff
@@ -414,12 +414,12 @@ static Image findBestImage(java.util.List icons, int width, int height) {
                     //larger than 1/scaleFactor
                     double scaleDivider = Math.ceil(1.0 / scaleFactor);
                     scaleFactor = 1.0 / scaleDivider;
-                    adjw = (int)Math.round((double)iw / scaleDivider);
-                    adjh = (int)Math.round((double)ih / scaleDivider);
+                    adjw = (int)Math.round(iw / scaleDivider);
+                    adjh = (int)Math.round(ih / scaleDivider);
```

Comment:
Same comment here about the old code being clearer.

Reply:
`scaleDivider` is defined just 2 lines above as a `double`. I don't see how the cast helps here.

Reply:
Yes, this one is fine.